src/plugins/displayLayout/components/TelemetryView.vue

```diff
@@ -169,11 +170,12 @@ export default {
                 return;
             }
 
+            let value;
             if (this.item.format) {
-                return printj.sprintf(this.item.format, this.datum[this.valueMetadata.key]);
+                value = this.formatter.format(this.datum, this.valueMetadata);
```

Comment:
Why are `formatter` and `valueFormatter` two different things? They have the same interface, so we should be able to use them interchangeably?

Reply:
`formatter` is `CustomStringFormat` and `valueFormatter` is I think `TelemetryValueFormatter`

Reply:
Yep, but the `CustomStringFormat` has the same interface as `TelemetryValueFormatter` by design so that we can use them interchangeably.

Reply:
yeah, updated `CustomStringFormat` code so that it is similar in terms of use.